content/recipes/repl.md

```diff
@@ -0,0 +1,64 @@
+### Read-Eval-Print-Loop (REPL)
+
+REPL is a simple interactive environment that takes single user inputs, executes them, and returns the result to the user.
+REPL feature lets you inspect your dependency graph and call methods on your providers (and controllers) directly from your terminal.
+
+#### Usage
+
+The REPL functionality is enabled when you call `repl` over some root NestJS module like so:
+
+```typescript
+@@filename(repl)
+import { repl } from '@nestjs/core';
+import { AppModule } from './app.module';
+
+async function bootstrap() {
+  await repl(AppModule);
+}
+bootstrap();
+@@switch
+import { repl } from '@nestjs/core';
+import { AppModule } from './app.module';
+
+async function bootstrap() {
+  await repl(AppModule);
+}
+bootstrap();
+```
+
+And then start the app just like the usual:
+
+```bash
+$ npm run start --entryFile repl
+```
+
+> info **Hint** `repl` returns a [Node.js REPL server](https://nodejs.org/api/repl.html), thus you can customize it as you wish.
+
+#### Demo
+
+<a href="https://asciinema.org/a/503623" target="_blank"><img src="https://asciinema.org/a/503623.svg" width="636"/></a>
```

Comment:
I didn't succeed on embedding the player here but you can try it out: https://asciinema.org/a/503623 & https://asciinema.org/docs/embedding